src/generators/dumbComponent.js

```diff
@@ -14,12 +14,14 @@ class DumbComponent extends GeneratorBlueprint {
 
   generatorArgs() {
     const creationPath = this.settings.getSetting('dumbPath');
```

Comment:
I would consider wrapping generated files in folders to be a global project setting so you can leave it in settings which gets injected into the Generator and access it there directly.